webapp/channels/src/components/admin_console/permission_schemes_settings/permissions_tree/permissions_tree.tsx

```diff
@@ -213,20 +213,57 @@ export default class PermissionsTree extends React.PureComponent<Props, State> {
         const sharedChannelsGroup = this.groups[9];
         const customGroupsGroup = this.groups[10];
 
-        if (config.EnableIncomingWebhooks === 'true' && !integrationsGroup.permissions.includes(Permissions.MANAGE_INCOMING_WEBHOOKS)) {
-            integrationsGroup.permissions.push(Permissions.MANAGE_INCOMING_WEBHOOKS);
+        if (config.EnableIncomingWebhooks === 'true') {
+            const incomingWebhookGroup = {
+                id: 'manage_incoming_webhooks_group',
+                permissions: [
+                    Permissions.MANAGE_OWN_INCOMING_WEBHOOKS,
+                    Permissions.MANAGE_INCOMING_WEBHOOKS,
```

Comment:
Is this the old Permission? `Permissions.MANAGE_INCOMING_WEBHOOKS` ? Do we need to add the others?